examples/keygen/keyimport.c

```diff
@@ -77,6 +110,15 @@ int TPM2_Keyimport_Example(void* userCtx, int argc, char *argv[])
         else if (XSTRCMP(argv[argc-1], "-xor") == 0) {
             paramEncAlg = TPM_ALG_XOR;
         }
+        else if (XSTRCMP(argv[argc-1], "-pem") == 0) {
```

Comment:
`-pem` is in here twice. Looks like copy/paste issue. This one should be deleted.